Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -1045,6 +1048,10 @@ def try_to_parse_directive(self, optname, args, kwds, pos):
                 raise PostParseError(pos,
                     'The %s directive takes one compile-time string argument' % optname)
             return (optname, directivetype(optname, str(args[0].value)))
+        elif directivetype is Ellipsis:
+            # signal to pass things on without processing
+            return (optname, (args,
+                              dict([(key.value, value) for key, value in kwds.key_value_pairs])))
```

Comment:
I've seen this code so often by now that I wonder if `DictNode` should simply grow a method for it.